src/app/components/pages/LogIn.tsx

```diff
@@ -182,21 +184,19 @@ export const EmailPasswordInputs =({setEmail, setPassword, validEmail, validPass
         </FormGroup>
 
         <FormGroup className="mb-0">
-            {displayLabels && (<><Label htmlFor="password-input">Password</Label>
+            {displayLabels && (<><Label htmlFor="current-password">Password</Label>
             <span id={`password-help-tooltip`} className="icon-help ml-1" />
             <UncontrolledTooltip target={`password-help-tooltip`} placement="bottom">
             {PASSWORD_REQUIREMENTS}
             </UncontrolledTooltip></>)}
-            <Input
-                id="password-input" autoComplete="current-password" type="password" name="password" placeholder="Password"
```

Comment:
There are multiple parameters that were present before, but are missing now, such as `placeholder`, `required`, `aria-describedby`, etc.

Reply:
required is included by default in the Password component so not necessary as a prop, aria-describedby is no longer needed (due to no feedback) but is still an optional prop for Password as it is used elsewhere. Placeholder change was intentional as it looks better, although I realised I also need to remove it from the email field on login page, so fixed that now.